pkgs/os-specific/linux/tailor-gui/default.nix

```diff
@@ -46,6 +46,11 @@ stdenv.mkDerivation {
     gtk4
   ];
 
+  postFixup = ''
+    wrapProgram $out/bin/tailor_gui \
+      --set GDK_BACKEND "x11"
+  '';
```

Comment:
suggestion: leave a comment linking to https://github.com/AaronErhardt/tuxedo-rs/issues/46,
with a `TODO` to remove this when it has been fixed.